impeller/entity/entity_pass.cc

```diff
@@ -124,10 +124,43 @@ std::optional<Rect> EntityPass::GetElementsCoverage(
 
       std::optional<Rect> unfiltered_coverage =
           GetSubpassCoverage(subpass, std::nullopt);
+
+      // If the current pass elements have any coverage so far and there's a
+      // backdrop filter, then incorporate the backdrop filter in the
+      // pre-filtered coverage of the subpass.
+      if (result.has_value() && subpass.backdrop_filter_proc_) {
+        std::shared_ptr<FilterContents> backdrop_filter =
+            subpass.backdrop_filter_proc_(FilterInput::Make(result.value()),
+                                          subpass.xformation_,
+                                          Entity::RenderingMode::kSubpass);
+        if (backdrop_filter) {
+          auto backdrop_coverage = backdrop_filter->GetCoverage({});
+          backdrop_coverage->origin += result->origin;
```

Comment:
Should this line be under the `if` on the line below?

Reply:
yeah this could be placed just above `unfiltered_coverage = backdrop_coverage`